### ISSUE TITLE
[Feature Request] the ability to delete customization from  the etc overlay

### ISSUE THREAD

**Issue by miki223** [ValveSoftware/SteamOS#1163]
i do not know how many times we have had people forget their root password & either have to wipe user data factory reset or reimage or use some other hacky  workaround but that is annoying & possibly time consuming so it would be nice if Steam OS at least on deck supported this
i know magisk does on Android all overlay modifications are disabled by entering safe mode

**Comment by bertogg** [comment]
You can do this by removing the files directly from `/var/lib/overlays/etc/upper/` and rebooting the system.

**Comment by miki223** [comment]
> You can do this by removing the files directly from `/var/lib/overlays/etc/upper/` and rebooting the system.

yes i know but would be nice to have a button that does this kinda annoying for folks without a keyboard & recovery media 
though there was this one exception we had where we were able to get away without doing this  the person who forgot their sudo password did it before they updated so we were able to boot the other rauc slot set their password there & upgrade replacing the old var partition in the process

**Comment by miki223** [comment]
the other hacky workaround we used is the old bash as init trick 
but that seems like it is broken currently
i am very curious why
seen it on other distros but never figured it out
guess i cannot mess with it & find out as i am using Nix OS

**Comment by miki223** [comment]
using init=/bin/bash -c  “command“ does work but i have only gotten folks to test it with fsck  /dev/nvme0p8 -y
which hung on the first yesno  prompt 
because of the old corrupted home issue systemd does not seem to run it by itself in this edge case
but the person i tested this hack with luckily had a suitable USB drive to use the recovery image
ran fsck from there & everything worked fine after
